apache/zeppelin
# [ZEPPELIN-4320] fix packages to bundle list

### What is this PR for?
Enabled Helium packages are not getting installed/bundled on zeppelin startup. Instead getting bundled later on paragraph with bundle dependency are loaded.


### What type of PR is it?
Bug Fix

### Todos

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-4320

### How should this be tested?
* Enable some helium packages
* Remove helium cache folder(local-repo)
* Restart zeppelin and verify, enabled packages are getting bundled on zeppelin server start

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update?
* Is there breaking changes for older versions?
* Does this needs documentation?
